qa/tasks/mon_thrash.py

```diff
@@ -60,7 +60,7 @@ class MonitorThrasher:
     freeze_mon_duration: how many seconds to freeze the mon (default: 15)
     scrub               Scrub after each iteration (default: True)
 
-    Note: if 'store-thrash' is set to True, then 'maintain-quorum' must also
```

Comment:
wouldn't it be simpler to just fix https://github.com/ceph/ceph/blob/master/qa/suites/rados/monthrash/thrashers/force-sync-many.yaml#L11 to use `store_thrash` and fix this comment?

Reply:
agreed!

Reply:
Updated. Please check again.